mesheryctl/internal/cli/root/model/import.go

```diff
@@ -175,13 +175,11 @@ func sendToAPI(data []byte, name string, dataType string) error {
 
 	var formFile io.Writer
 	if dataType == "file" {
-		formFile, err = writer.CreateFormFile("file", filepath.Base(name))
+		formFile, _ = writer.CreateFormFile("file", filepath.Base(name))
 	} else {
-		formFile, err = writer.CreateFormField("dir")
+		formFile, _ = writer.CreateFormField("dir")
 	}
-	if err != nil {
 
-	}
```

Comment:
Why are we removing the error handling

Reply:
In different PR added them.

Reply:
When I write support for OCI should be done by today if I get the OCI from Nitish then I'll update this to support it too. Then only I would write the meshkit error for this.

Reply:
@Yashsharma1911